src/gh_pr/ui/interactive.py

```
@@ -0,0 +1,635 @@
+        margin: 1 0;
+    }
+    """
+
+    # Define available filters with descriptions
+    FILTERS = {
+        "all": "All Comments",
+        "unresolved": "Unresolved Only",
+        "resolved_active": "Resolved (Active)",
+        "unresolved_outdated": "Unresolved (Outdated)",
+        "current_unresolved": "Current Unresolved",
+    }
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)